doc/api/api_changes.rst

```diff
@@ -15,13 +15,140 @@ For new features that were added to matplotlib, please see
 Changes in 2.0.0
 ================
 
+
+
+Deprecation and removal
+-----------------------
+
 Color of Axes
--------------
+`````````````
 
 The ``axisbg`` and ``axis_bgcolor`` properties on ``Axes`` have been
 deprecated in favor of ``facecolor``.
 
 
+GTK and GDK backends deprecated
+```````````````````````````````
+The untested and broken GDK and GTK backends have been deprecated.
+These backends allows figures to be rendered via the GDK api to
+files and GTK2 figures. They are untested, known to be broken and
+use have been discouraged for some time. The `GTKAgg` and `GTKCairo` backends
+provide better and more tested ways of rendering figures to GTK2 windows.
+
+WX backend deprecated
+`````````````````````
+The untested WX backend has been deprecated.
```

Comment:
Maybe change "untested" to "standalone", to help distinguish it from the WxAgg backend.  The "untested" aspect is stated later.